app/Http/Controllers/ProceduresController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Procedure;
+
+class ProceduresController extends Controller
+{
+    /**
+	 * Display a listing of the resource
+	 *
+	 * @return Response
+	 */
+	public function index()
+	{
+
+		$response = [];
+		$procedures = Procedure::with('items')->get();
```

Comment:
I think `Procedure::all()` should do it, no need to do a `with` to be able to use the relation